src/aiida/cmdline/utils/decorators.py

```diff
@@ -259,7 +265,14 @@ def wrapper(wrapped, _, args, kwargs):
 
         template = templates.env.get_template('deprecated.tpl')
         width = 80
-        echo.echo(template.render(msg=wrap(message, width - 4), width=width))
+        if version is not None:
+            version_object = Version(version)
+            # we print the version always in X.Y.Z format, e.g. "1.2" -> "1.2.0", "1" -> "1.0.0"
+            long_version = f'{version_object.major}.{version_object.minor}.{version_object.micro}'
+            extended_message = f'{message} (this will be removed in v{long_version})'
+        else:
+            extended_message = f'{message}'
```

Comment:
Now that a default is defined for `version` it doesn't really make sense to deal with the `None` case. Instead, we should validate the version provided. Something like:
```suggestion
        from packaging.version import InvalidVersion
        
        try:
            version_object = Version(version)
        except (TypeError, InvalidVersion) as exception:
            raise ValueError(f'`{version}` is not a valid value for `version`') from exception

        extended_message = f'{message} (this will be removed in v{version_object.major}.{version_object.minor})'
```

Reply:
Right